static/app.js

```diff
@@ -42,6 +41,46 @@ $(function() {
         console.log('Geolocation not enabled in this browser')
     }
 
+    let timeoutId;
+
+    $('#term').on('keyup', ()=> {
+        if ($('#search_type').val() == 'by keyword') {
+            clearTimeout(timeoutId);
+            timeoutId = setTimeout(runAutocomplete, 500)
+        } 
+    });
+
+    async function runAutocomplete(term) {
+        $('#auto').remove()
+        term = $('#term').val()
+        try {
+            const response = await axios({
+                url: `${base_url}autocomplete`,
+                method: "GET",
+                params: {
+                    query: term
+                }
+            })
+            $('#term').after('<div id="auto"></div>')
```

Comment:
Could this be part of the render?

Reply:
When discussing you also suggested to investigate possible solution here, so I've tried a different approach, which is simply adding the auto div to home.html and then emptying it and filling it as necessary.  What do you think?